### PR TITLE
Switch from deprecated float3-style names to SIMD3<Float> in overlays

### PR DESCRIPTION
**Pull request by stephentyrone**

### CODE REVIEW

**Comment by stephentyrone** [comment]
@swift-ci please test

**Comment by stephentyrone** [comment]
@milseman This is mostly mechanical, but please give a quick scan for anything out of place.

File: stdlib/public/Darwin/SpriteKit/SpriteKit.swift [hunk 1]
```diff
@@ -62,7 +62,7 @@ extension SKWarpGeometryGrid {
   /// - Parameter rows: the number of rows to initialize the SKWarpGeometryGrid with
   /// - Parameter sourcePositions: the source positions for the SKWarpGeometryGrid to warp from
   /// - Parameter destinationPositions: the destination positions for SKWarpGeometryGrid to warp to
-  public convenience init(columns: Int, rows: Int, sourcePositions: [simd.float2] = [float2](), destinationPositions: [simd.float2] = [float2]()) {
+  public convenience init(columns: Int, rows: Int, sourcePositions: [SIMD2<Float>] = [SIMD2<Float>](), destinationPositions: [SIMD2<Float>] = [SIMD2<Float>]()) {
```
Comment (benrimmington) [inline stdlib/public/Darwin/SpriteKit/SpriteKit.swift]: Can the default arguments use empty array literals?

```suggestion
  public convenience init(columns: Int, rows: Int, sourcePositions: [SIMD2<Float>] = [], destinationPositions: [SIMD2<Float>] = []) {
```

**Comment by benrimmington** [comment]
@stephentyrone By the way, the `pointwiseMin` and `pointwiseMax` SIMD operations aren't in the Swift 5.1 branch yet.

File: stdlib/public/Darwin/SpriteKit/SpriteKit.swift (hunk 1 above)
Comment (stephentyrone) [reply stdlib/public/Darwin/SpriteKit/SpriteKit.swift]: They could, but I'd prefer to keep this PR restricted to *just* updating the type names.

**Pull request merged**